code/client/src/pages/Show/Swap.jsx

```diff
@@ -208,7 +208,7 @@ const Swap = ({ address }) => {
       return
     }
     const erc20Tracked = (wallet.trackedTokens || []).filter(e => e.tokenType === ONEConstants.TokenType.ERC20)
-    const trackedTokens = [harmonyToken, ...withKeys(DefaultTrackedERC20(network)), ...(erc20Tracked || [])]
+    const trackedTokens = [harmonyToken, ...(erc20Tracked || []), ...withKeys(DefaultTrackedERC20(network))]
```

Comment:
I think default ones should go first, so the tracked ones can override them if there is any update

Reply:
The issue is the tracked token might not have icons from the sushiswap cloudinary, e.g. 1aave

Reply:
Let's move icons to a GCP storage bucket later